docs/modeling/connectors/ootb/email.md

```diff
@@ -24,6 +24,9 @@ The following are the parameters that can be passed to the email connector as in
 | `text` | A plain text email body | String | No |
 | `html` | An HTML email body | String | No |
 | `charset` | Define the *charset* of the email | String | No | 
+| `nodeId` | The node ID of the file to attach to the email | String | No |
+| `uri` | The URI of the file to attach to the email | String | No |
+| `files` | A [file](../../modeling/files.md) uploaded in a process and set as a process variable or uploaded as part of a form or another connector to attach to the email | File | No |
```

Comment:
```suggestion
| `files` | A [file](../../files.md) uploaded in a process and set as a process variable or uploaded as part of a form or another connector to attach to the email | File | No |
```

Reply:
Yes, that is the only thing I spotted here

Reply:
Thank you for the good spot @nazarethjim !